schema/DanmNet.yaml

```diff
@@ -36,6 +36,12 @@ spec:
     # A Virtual Function belonging to this Physical Function is taken-up in case NetworkType is set to SRIOV.
     # OPTIONAL - STRING
     host_device: ## PARENT_DEVICE_NAME ##
+    # Name of a network Device Plugin resource pool
+    # The device_pool parameter generally represents the base resource name of the Kubernetes Devices connected to this network.
+    # Specifically for DanmNets with "NetworkType: sriov" this option is mandatory, and it represents a pool of VFs Pods connecting to this network will get one provisioned from.
+    # container_prefix must be ommited, Intel SR-IOV CNI generates unique interface name automatically 
```

Comment:
I commented at the beginning of this PR too, that container_prefix option is working with SR-IOV CNI, and it is DANM which generates the unique
host_device is the parameter which is ignored from now on in a NetworkType: sriov DanmNet
Ignorance of a specific attribute should be documented in the description of the attribute itself

Reply:
Applied.
Thanks. This line remaied there due to editing failure, the intence was to remove it.